src/components/forms/ReserveAppointmentForm.jsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react';
+import axios from 'axios';
+
+const ReservationForm = () => {
```

Comment:
- [ ] When I try to reserve a car, this happens:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/ac72657b-c68c-4038-ba52-b8e82d39c3cf)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/f73674ad-5a89-4c67-8389-141b219658da)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/67f1d465-a9b7-4566-9d35-0321d26b0ff7)

It seems the car id and user id that are being sent to the server are null for some reason.

Also, it is worth looking into the fact that when reserving, you cannot select the car.




For context here are the cookies:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/4348d26a-35ce-422e-b82e-140765bab19c)

Local Storage:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/01345bb1-646a-4036-8141-4fb54b73402f)


Kindly, fix this bug so your app works as intended. 👍 

Here is an extract of the requirements for teams of 3:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/892b6564-1993-48dd-a701-a8c6219f85d5)

Reply:
Hello @Meltrust Sir, Thanks for giving us useful insights about website, however The Reservations and backend parts are by our partner who just left our team because he had some personal issue, However he convey this msg to Student Success and we also, here our kanban board link: https://github.com/users/MasumaJaffery/projects/4  and here my email req:  https://mail.google.com/mail/u/0/#sent/KtbxLxgGCTrskrjPqhJpqPtDDgPDFZHhpg  However we try our best!.

Reply:
> Hello @Meltrust Sir, Thanks for giving us useful insights about website, however The Reservations and backend parts are by our partner who just left our team because he had some personal issue, However he convey this msg to Student Success and we also, here our kanban board link: https://github.com/users/MasumaJaffery/projects/4 and here my email req: https://mail.google.com/mail/u/0/#sent/KtbxLxgGCTrskrjPqhJpqPtDDgPDFZHhpg However we try our best!.

Hey @MasumaJaffery, Noted. Thanks for the clarification.